components/Organization/OrganizationCard.tsx

```diff
@@ -0,0 +1,20 @@
+import { Card } from 'react-bootstrap';
+
+import { Organization, OrganizationTypeName } from '../../models/Organization';
+
+export const OrganizationCard = ({
+  name,
+  description,
+  type,
+  logo,
+}: Organization) => (
+  <Card>
+    <Card.Img src={logo?.uri} alt={logo?.name} title={logo?.name} />
+    <Card.Body>
+      <Card.Title>{name}</Card.Title>
+      <Card.Subtitle>{OrganizationTypeName[type]}</Card.Subtitle>
+      <hr></hr>
+      <Card.Text>{description}</Card.Text>
```

Comment:
```suggestion
      <Card.Text className="border-top my-2 pt-2">{description}</Card.Text>
```
1. `<hr />` 是自闭合标签：https://developer.mozilla.org/zh-CN/docs/Web/HTML/Element/hr
2. 可以直接用 Bootstrap 工具类画样式